app/models/service_ansible_playbook.rb

```diff
@@ -32,6 +33,10 @@ def execute(action)
     add_resource!(new_job, :name => action)
   end
 
+  def retireable?
+    !retain_resources_on_retirement?
```

Comment:
So discussed more with @tinaafitz and this does not feel correct.  If you ask the service instance if it can be retired the answer should not be solely used to drive if child resources are retired, it should be answering the question for it self.  Another way of looking at it would be if the UI was using this method to determine if the service itself could be retired.  I think we want the response to be true regardless of the child-items.

Seems like the `retain_resources_on_retirement` is doing more what we want and this method is not needed.

Reply:
I was under the impression that at the end of the conversation I and you and Madhu had in the kitchen that the proposed solution then was to use this retireable? check from the CiFeatureMixin to determine whether the service was retireable. If we're not using this approach I think we should have the discussion that Jason kinda started on last week about how checking to see if something is retireable requires both a general check like those used by the SupportsFeatureMixin and something more specific because since this issue is related it'd be easier to have that discussion sooner.